### ISSUE TITLE
IOS it doesn't work when app is killed(Android is working)

### ISSUE THREAD

**Issue by FetFrumos** [rekab-app/background_locator#309]
I using last version plugin background_locator: ^1.6.12. But example doesn't work in IOS when app is killed. it works in android perfectly.
I followed all the instructions in the wiki. What could be the reason? Any advice - I'll be grateful. By the way, the example can be updated.

**Comment by FetFrumos** [comment]
any news? anybody here?

**Comment by treacks5** [comment]
Same here. 
Potentially same issue #291 🤷‍♂️

Details : 

I have a flutter iOS mobile app that display a push notification (using 'https://fcm.googleapis.com/fcm/send') every time the location callback is trigger.

I receive the location notification on foreground and background, but not when I manually kill the app.
What is curious is that I can see the native navigation icon on the top left (next to the iPhone time) even when the app is killed. So I suppose that the background_locator location service is working great but it is just that the callback is not triggered.

background_locator version is 1.6.12

**Comment by Mayb3Nots** [comment]
When you terminate the app did you test after moving the location about 500m away? Because the way IOS works is it will only send the device's location after ~500m location change when the app is terminated.  If it doesn't move it won't trigger.

**Comment by FetFrumos** [comment]
yes, after the application is killed - it does not work, on any melt. @Mayb3Nots  do you have a working example?
